pkg/controller/storagecluster/quickstart_constants.go

```diff
@@ -0,0 +1,201 @@
+  description: Learn how to configure OpenShift Container Storage to meet your deployment
+    needs.
+  prerequisites: ["Getting Started with OpenShift Container Storage"]
+  introduction: In this tour you will learn about the various configurations available
+    to customize your OpenShift Container Storage deployment.
+  tasks:
+  - title: Expand the OCS Storage Cluster
+    description: |- 
+      When we install the OCS operator we created a storage cluster, chose
+      the cluster size, provisioned the underlying storage subsystem, deployed necessary
+      drivers, and created the storage classes to allow the OpenShift users to easily
+      provision and consume storage services that have just been deployed
+      
+      When the capacity of the cluster is about to runout we will notify you.
+      **To expand the OCS storage cluster follow these steps:**
+      
+      1. Go to installed operators page and click on **OpenShift Container Storage** 
+      2. Go to storage cluster tab
+      3. Click on the **3 dots icon** 
+      4. Click on add capacity
+      5. Use the expand cluster modal if your capacity is about to runout.
+    review:
+      instructions: |-
+        ####  To verify that you have expanded your storage cluster.
+
+        Did you expand your cluster?
+      failedTaskHelp: This task isn’t verified yet. Try the task again.
+    summary:
+      success: You have expanded the OpenShift Container Storage operator!
+      failed: Try the steps again.
+  - title: Bucket Class Configuration
+    description: |-
+      **Bucket class**
+      Bucket class policy determine the bucket's data location. Its set of policies which would apply to all buckets (OBCs) created with the specific bucket class.
+
+      Data placement capabilities area built as a multi-tier structure, here are the tiers
+
+      **Spread Tier** - list of backing-stores, aggregates the storage of multiple stores.
+
+      **Mirroring Tier** - list of spread-Tiers, async-mirroring to all mirrors, with locality optimization.
+
+      **Create a new Bucket class:**
+
+      1. Go to installed operators page and click one OpenShift Container Storage,
```

Comment:
Typo: Go to the Installed Operators page and click "on" ...

Reply:
fixed